src/execution/vm/llvm_engine.cpp

```diff
@@ -948,29 +951,31 @@ void LLVMEngine::CompiledModuleBuilder::Simplify() {
 }
 
 void LLVMEngine::CompiledModuleBuilder::Optimize() {
-  llvm::legacy::PassManager module_passes;
   llvm::legacy::FunctionPassManager function_passes(llvm_module_.get());
 
-  // Add the appropriate TargetLibraryInfo and TargetTransformInfo
-  auto target_library_info_impl = std::make_unique<llvm::TargetLibraryInfoImpl>(target_machine_->getTargetTriple());
+  // Add the appropriate TargetTransformInfo.
   function_passes.add(llvm::createTargetTransformInfoWrapperPass(target_machine_->getTargetIRAnalysis()));
-  module_passes.add(new llvm::TargetLibraryInfoWrapperPass(*target_library_info_impl));
-  module_passes.add(llvm::createTargetTransformInfoWrapperPass(target_machine_->getTargetIRAnalysis()));
 
-  // Build up optimization pipeline
+  // Build up optimization pipeline.
   llvm::PassManagerBuilder pm_builder;
   pm_builder.OptLevel = 3;
```

Comment:
Perhaps in `CompiledModuleBuilder`:

`constexpr static const unsigned OPT_LEVEL = 3;`